pkg/workflow/argo.go

```diff
@@ -498,6 +621,28 @@ func defaultExecutor() v1alpha12.Template {
 	}
 }
 
+func defaultReversalExecutor() v1alpha12.Template {
+	return v1alpha12.Template{
+		Name:               helmReleaseReversalExecutor,
+		ServiceAccountName: workflowServiceAccountName(),
+		Inputs: v1alpha12.Inputs{
+			Parameters: []v1alpha12.Parameter{
+				{
+					Name: "helmrelease",
+				},
+			},
+		},
+		Executor: &v1alpha12.ExecutorConfig{
+			ServiceAccountName: workflowServiceAccountName(),
+		},
+		Resource: &v1alpha12.ResourceTemplate{
+			// SetOwnerReference: true,
+			Action:   "delete",
```

Comment:
I may need to add an equivalent of Success/Failure condition for delete action

Reply:
Can you try something like this,
```
successCondition: func() string {
   // lookup helm release using the helmv3 go-client package
  if len(lookupOutput) == 0 {
    return "true"
  }
  return "false"
}()
```
I am not sure if this will work or not but worth trying I suppose